core/src/lib.rs

```diff
@@ -22,6 +24,8 @@ mod constraints;
 mod events;
 mod gravity;
 mod layers;
+#[cfg(feature = "scenes")]
+mod pending_collision;
```

Comment:
Name the module after the the feature flag

Reply:
Done!